
<launch>

 <node pkg="learning_topic" type="person_publisher" name="robot_2001998" output="screen" />
 <node pkg="learning_topic" type="person_subscriber" name="JiangJunjie" output="screen" /> 

</launch>
